src/app/104-hao/104-hao.component.ts

```diff
@@ -52,10 +53,11 @@ export class Hao_104_Component implements OnInit, OnChanges, DoCheck {
     private dialogService: ConfirmationDialogsService,
     public commonAppData: dataService,
     private _callServices: CallServices,
+    private sessionstorage:sessionStorageService,
     private httpServices: HttpServices
   ) {
-    if (sessionStorage.getItem("service") != undefined) {
-      this.outboundFor = sessionStorage.getItem("service");
+    if (this.sessionstorage.getItem("service") != undefined) {
```

Comment:
Consider using camelCase for variable names  to follow typescript naming conventions